src/esp/assets/GenericMeshData.cpp

```diff
@@ -44,16 +44,15 @@ Magnum::GL::Mesh* GenericMeshData::getMagnumGLMesh() {
   return &(renderingBuffer_->mesh);
 }
 
-void GenericMeshData::setMeshData(
-    Corrade::Containers::Optional<Magnum::Trade::MeshData>& meshData) {
+void GenericMeshData::setMeshData(Magnum::Trade::MeshData& meshData) {
```

Comment:
```suggestion
void GenericMeshData::setMeshData(Magnum::Trade::MeshData&& meshData) {
```

I really meant accepting a r-value here, to convey that there's an ownership transfer happening inside.

Reply:
Oops.